eZ/Bundle/EzPublishCoreBundle/Resources/config/routing.yml

```diff
@@ -96,6 +97,16 @@ services:
         tags:
             - { name: kernel.event_subscriber }
 
+    ezpublish.request_index_listener:
+        class: %ezpublish.request_index_listener.class%
+        arguments:
+            - @ezpublish.config.resolver
```

Comment:
(Note: the service definition should be updated aswell to remove extra dependencies)